### PR TITLE
Update RF team

### PR DESCRIPTION
**Pull request by nali**
Also add consistency to development team paragraphs.

### CODE REVIEW

**Comment by javan** [comment]
Welcome! ☺️

File: orgchart.md [hunk 1]
```diff
@@ -64,11 +64,11 @@ Core Product works on new features for Basecamp together with the product design
...
 
 #### Research & Fidelity (RF)
 
-Our [rich text editor Trix](https://trix-editor.org) and our [front-end framework Turbolinks](https://github.com/turbolinks/turbolinks) were both born from RF. They work on deep research projects like those two, but also on all manners of UI fidelity in general. That means a lot of JavaScript, but also integrating that with our Ruby and native code. **Javan Makhmali** is team lead and works with Sam Stephenson.
+Our [rich text editor Trix](https://trix-editor.org) and our [front-end framework Turbolinks](https://github.com/turbolinks/turbolinks) were both born from RF. They work on deep research projects like those two, but also on all manners of UI fidelity in general. That means a lot of JavaScript, but also integrating that with our Ruby and native code. **Javan Makhmali** is the team lead and works with Sam Stephenson and Nabeelah Ali.
```
Comment (kaspth) [inline orgchart.md:71]: If we want a little extra consistency: I think our names are sorted alphabetically by last name 😄 Looks like it in Core Product.

Comment (nali) [reply orgchart.md:71]: I think you're onto something. OMG members are definitely sorted in that way, and SIP _almost_ gets there but slips up with Jorge.

Comment (rosa) [reply orgchart.md:71]: Ooh, looks like SIP are sorted by order of arrival, like the Ops teams 😊

Comment (kaspth) [reply orgchart.md:71]: Well, clearly we can all agree Core Product is the best team and thus the example to follow 😂

Ok, didn't spot there were different mixes.

**Pull request merged**